tests/testthat/test-add_quantiles.R

```diff
@@ -0,0 +1,281 @@
+#' @title Specifications add_quantiles.R
+#' @section Last updated by: Tim Treis
+#' @section Last update date: 16-Nov-2021
+
+# Specifications ---------------------------------------------------------------
+
+#' T1. No error when quantile lines are added to a plot.
+#' T1.1 No error when the default parameters are used.
+#' T1.2 No error when `quantiles` is a single number.
+#' T1.3 No error when `quantiles` is a numeric vector.
+#' T1.4 No error when `linetype` is a non-default character string.
+#' T1.5 No error when `linetype` is `mixed`.
+#' T1.6 No error when `linecolour` is a non-default character string.
+#' T1.7 No error when `linecolour` is `strata`.
+#' T2. The quantile lines are y-axis-transformation dependent.
+#' T2.1 No error when the default `surv` option is used.
+#' T2.2 No error when a function is passed as a string.
+#' T2.3 No error when a `.Primitive` function is used.
+#' T2.4 No error when a custom function is used.
+#' T3. If any parameter is specified incorrectly, defaults are used instead.
```

Comment:
This seems to be a repeat of earlier tests. I like this general T3. and then putting everything below it regarding parameter choices.

Use argument instead of parameter